drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/table/utilities/ColumnUtilities.java

```diff
@@ -145,13 +145,22 @@ public static boolean canAcceptOtherwiseValues(final BaseColumn column) {
 
         final String[] values = ListSplitter.split("'", true, valueList);
 
-        return Stream.of(values).filter(value -> isValueValidForType(value,
+        return Stream.of(values).filter(value -> isValueValidForType(parseActualValue(value),
                                                                      convertToTypeSafeType(fieldType)))
                 .toArray(String[]::new);
     }
 
+    private String parseActualValue(final String value) {
+        if (value != null && value.indexOf("=") > 1) {
```

Comment:
I think this should be `> 0` since a value list item of `1=One` does not work.

Reply:
@manstis Well spotted and now fixed.